Sources/UIImage+Sweet.swift

```diff
@@ -3,6 +3,26 @@
 
     public extension UIImage {
 
+        /// Creates a QR code from a string.
+        ///
+        /// - Parameter string: Text to be the QR Code content
+        /// - Returns: image QR Code image
+        public static func imageQRCode(for string: String) -> UIImage {
```

Comment:
Not sure. It's all Apple's code, really. There's nothing to refactor I don't think. We could roll a few example images from some strings and see that those images match to the test ones at some point, just in case.... I'd rather move a bit faster today tho. Issue?